ssa/manager_wait.go

```diff
@@ -129,7 +145,7 @@ func (m *ResourceManager) WaitForSet(set object.ObjMetadataSet, opts WaitOptions
 				errors = append(errors, builder.String())
 			}
 		}
-		return fmt.Errorf("timeout waiting for: [%s]", strings.Join(errors, ", "))
+		return fmt.Errorf("%s: [%s]", msg, strings.Join(errors, ", "))
```

Comment:
Does the error contains the objects that have failed? what does the message look like, can you add a test for it please.